base/utils/rpm.go

```diff
@@ -96,3 +98,11 @@ func (n Nevra) EVRAStringE(showEpoch bool) string {
 func (n Nevra) EVRAString() string {
 	return n.EVRAStringE(false)
 }
+
+func (n Nevra) Cmp(other *Nevra) int {
+	ret := strings.Compare(n.Name, other.Name)
+	if ret != 0 {
+		return ret
+	}
```

Comment:
I'm just curious, what is the purpose of comparing rpm names?

Reply:
The aim is to sort NEVRAs, rpmlib can only compare EVRA, so the first thing you sort/compare is name, then EVRA.